Jenkinsfile

```diff
@@ -192,6 +193,9 @@ pipeline {
                     ].join(" ")
 
                     skipRemainingStages = utils.verifyChanges(paths)
+
+                    def openCLPaths = ['lib/stan_math/stan/math/opencl', 'lib/stan_math/test/unit/math/opencl'].join(" ")
+                    skipOpenCL = utils.verifyChanges(openCLPaths)
```

Comment:
This should just check if there are  changes in `src/stan/model/indexing`, we dont really care about changes in math here.